### PR TITLE
Add test for packages check-update check

### PR DESCRIPTION
**Pull request by shubhamsg199**
**Test Results:**

**Satellite:**

 ```
$ pytest -v --ansible-host-pattern server --ansible-user=root --ansible-inventory testfm/inventory tests/test_packages.py -k test_positive_packages_check_update
=============================================== test session starts ===========================================================
platform linux -- Python 3.8.8, pytest-3.6.1, py-1.10.0, pluggy-0.6.0 -- /home/sganar/foreman/fm/bin/python
cachedir: .pytest_cache
ansible: 2.9.20
rootdir: /home/sganar/foreman/testfm, inifile:
plugins: ansible-2.2.3
collected 4 items / 3 deselected                                                                                                                                                                                                          

tests/test_packages.py::test_positive_packages_check_update PASSED                                                                                                                                                                  [100%]

======================================= 1 passed, 3 deselected in 230.27 seconds =========================================
```

**Capsule:**

```
$ pytest -v --ansible-host-pattern capsule --ansible-user=root --ansible-inventory testfm/inventory tests/test_packages.py -k test_positive_packages_check_update
========================================== test session starts ==================================================
platform linux -- Python 3.8.6, pytest-3.6.1, py-1.10.0, pluggy-0.6.0 -- /home/sganar/foreman/fm/bin/python
cachedir: .pytest_cache
ansible: 2.9.20
rootdir: /home/sganar/foreman/testfm, inifile:
plugins: ansible-2.2.3
collected 4 items / 3 deselected                                                                                                                                                                                  

tests/test_packages.py::test_positive_packages_check_update PASSED                                                                                                                                          [100%]

=================================== 1 passed, 3 deselected in 422.46 seconds ========================================================
```

### CODE REVIEW

File: testfm/packages.py [hunk 1]
```diff
@@ -168,3 +168,28 @@ def is_locked(cls, options=None):
         result = cls._construct_command(options)
 
         return result
+
+    @classmethod
+    def check_update(cls, options=None):
+        """
+        Usage:
```
[the remark this answers is not in the document]
Comment (Gauravtalreja1) [reply testfm/packages.py:176]: Nope, I was saying about reverse condition, no metadata for subcommands, the same pattern is used for robottelo cli as well

File: testfm/packages.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
     install                       Install packages in an unlocked session
     update                        Update packages in an unlocked session
     is-locked                     Check if update of packages is allowed
+    check-update                  Check if update of packages is available
```
Comment (Gauravtalreja1) [inline testfm/packages.py:16]: Here help description seems different, seems to be entered manually, we should be following exact details and order for metadatas.

File: testfm/packages.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (shubhamsg199) [reply testfm/packages.py:176]: ah..got it, yeah we can do that

File: testfm/packages.py (hunk 2 above)
Comment (shubhamsg199) [reply testfm/packages.py:16]: Okay

**Review by Gauravtalreja1 — APPROVED** [review]
LGTM, Thanks for the PR.

@shubhamsg199 will just ask to include `:BZ: ` in test metadata, and @jameerpathan111 can review/merge.

**Review by jameerpathan111 — APPROVED** [review]

**Pull request merged**